onadata/libs/constants.py

```diff
@@ -35,4 +35,3 @@
 
 CAN_ADD_DATADICTIONARY = 'add_datadictionary'
 CAN_CHANGE_DATADICTIONARY = 'change_datadictionary'
-CAN_DELETE_DATADICTIONARY = 'delete_datadictionary'
```

Comment:
What was the reason for removing `CAN_DELETE_DATADICTIONARY`?

Reply:
Good catch :sweat_smile: